contributor-docs/adrs/adr-017-css.md

```diff
@@ -73,16 +73,16 @@ The other reasons for change are to utilise css variables for theming and improv
 Code example:
 
 ```jsx
-import classNames from './ActionList.module.css'
 import clsx from 'clsx'
+import styles from './ActionList.module.css'
```

Comment:
@siddharthkp we may need to look at using [this](https://gist.github.com/developit/3436591071d6b8f12dc87ca53c20dfa6) to process this file.

In which case the import will look more like:

```ts
import * as styles './ActionList.module.css'
```

which we can build a quick eslint rule for as well.

Reply:
oh very cool! 

> import * as styles './ActionList.module.css'

as long as this is compatible with rollup, storybook and gatsby, we're good!

(Our gatsby docs throw a warning right now if you add a component with css to drafts/index 😢, it's something we have to figure out)